### ISSUE TITLE
Update httpclient artifact

### ISSUE THREAD

**Issue by lewismc** [maxmind/GeoIP2-java#40]
Hi Folks,
There are newever versions of the Apache httpclient artifact which this library should consider updating to.
Clients consuming geoip2 have classloading issues due to geoip2 using an old httpclient version.
I'll see if I can get something together and submit a PR.

**Comment by oschwald** [comment]
We currently use the [Google HTTP Client Library](https://github.com/google/google-http-java-client), which is what is pulling in the old version of httpclient. We have considered moving to using Apache httpclient directly to help avoid issues like this, but the work for doing so isn't currently scheduled.

**Comment by lewismc** [comment]
That's a real shame... I will see if I can maybe even submit a patch for transition to httpclient

**Comment by oschwald** [comment]
I'm not sure if it will work, but have you tried excluding the old version of httpcient? For instance:

```
    <dependency>
        <groupId>com.maxmind.geoip2</groupId>
        <artifactId>geoip2</artifactId>
        <version>2.2.0</version>
        <exclusions>
          <exclusion>
            <groupId>org.apache.httpcomponents<</groupId>
            <artifactId>httpclient</artifactId>
          </exclusion>
        </exclusions> 
    </dependency>
```

**Comment by irmakozonay** [comment]
Excluding the old version worked for me.

**Comment by oschwald** [comment]
This is fixed by 2.8.0-rc1.
